Makefile

```diff
@@ -2,7 +2,7 @@ PACKAGE         ?= dcos-net
 VERSION         ?= $(shell git describe --tags)
 BASE_DIR         = $(shell pwd)
 ERLANG_BIN       = $(shell dirname $(shell which erl))
-REBAR            = $(shell pwd)/rebar3
+REBAR            = $(shell pwd)/bin/run.sh ./rebar3
```

Comment:
@urbanserj suggestion:
Instead of having a `run.sh`, can we just have `docker-image` and `docker` as a target.

`make docker-image` can build the docker image and `make docker` can do what `run.sh` does?

This will make it simple for developers to build without docker in case they don't wish to build and run within a docker container?

Reply:
I've added `docker`, `docker-image` and `docker-*` targets. So now you can run any makefile targets in docker, e.g. `make docker-compile docker-ct docker-dialyzer`. Could you please review this PR again?